app-editors/ghostwriter/ghostwriter-2.1.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+BDEPEND="dev-qt/linguist-tools:5"
+
+RDEPEND="
+	app-text/hunspell:=
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwebchannel:5
+	dev-qt/qtwebengine:5[widgets]
+	dev-qt/qtwidgets:5
+"
+DEPEND="${RDEPEND}"
+
+DOCS=( CREDITS.md README.md )
+
+src_prepare() {
+	default
+
+	sed -i -e "/^VERSION =/s/\$.*/${PV}/" ghostwriter.pro || die "failed to override version"
```

Comment:
Does this sed still serve a purpose? The version is already "VERSION = 2.1.1" in the .pro

From app-portage/iwdevtools:
```
 * SED: the following did not cause any changes
 *     sed -i -e "/^VERSION =/s/\$.*/${PV}/" ghostwriter.pro || die "failed to override version"
```

Reply:
The 2.1.2 bump that I did didn't throw this (w/ iwdevtools bashrc symlink setup), so it may still be required.